packages/core/src/components/popover/popover.tsx

```diff
@@ -381,10 +381,13 @@ export class Popover extends AbstractComponent<IPopoverProps, IPopoverState> {
         if (props.isModal && props.interactionKind !== PopoverInteractionKind.CLICK) {
             throw new Error(Errors.POPOVER_MODAL_INTERACTION);
         }
-        try {
-            React.Children.only(props.children);
-        } catch (e) {
-            throw new Error(Errors.POPOVER_ONE_CHILD);
+        if (typeof props.children === "object") {
```

Comment:
what's the reasoning for the change here? just ensuring we support non-JSX children in `Popover` without freaking out?